src/common_format.rs

```diff
@@ -124,14 +124,20 @@ impl Calendar {
 fn manage_calendar_dates<H>(
     calendars: &mut CollectionWithId<objects::Calendar>,
     file_handler: &mut H,
+    calendar_exists: bool,
 ) -> Result<()>
 where
     for<'a> &'a mut H: FileHandler,
 {
     let file = "calendar_dates.txt";
     let (reader, path) = file_handler.get_file_if_exists(file)?;
     match reader {
-        None => info!("Skipping {}", file),
+        None => {
+            info!("Skipping {}", file);
```

Comment:
I think this should be after the condition `if !calendar_exists {`
if none of the calendars exist we don't want to see that message

Reply:
Yep. Well seen.